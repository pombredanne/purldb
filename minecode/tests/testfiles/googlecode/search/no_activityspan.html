



<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >

 <meta name="ROBOTS" content="NOINDEX,NOARCHIVE">

 <link rel="icon" type="image/vnd.microsoft.icon" href="https://ssl.gstatic.com/codesite/ph/images/phosting.ico">


 <script type="text/javascript">




 var codesite_token = "ABZ6GAcm6E7sAQQpPKgGQG5puRbRLBTSeg:1402424806285";


 var CS_env = {"token": "ABZ6GAcm6E7sAQQpPKgGQG5puRbRLBTSeg:1402424806285", "profileUrl": "/u/102129653793910296719/", "projectName": null, "assetHostPath": "https://ssl.gstatic.com/codesite/ph", "domainName": null, "projectHomeUrl": null, "assetVersionPath": "https://ssl.gstatic.com/codesite/ph/3374407039186159233", "loggedInUserEmail": "tdruez@nexb.com", "relativeBaseUrl": ""};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);

 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();

 </script>


 <title>




 Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/3374407039186159233/css/core.css">

 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/3374407039186159233/css/ph_list.css" >




<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/3374407039186159233/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }


 .labels a:link { color: #080 }
 .labels a:visited { color: #080 }
 .labels a:active { color: #f00 }
 .name { margin-top: 2ex; font-size: 120% }


 </style>
</head>
<body class="">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function()
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">


 <span>



 <a href="#" id="multilogin-dropdown" onclick="return false;"
 ><u><b>tdruez@nexb.com</b></u> <small>&#9660;</small></a>


 | <a href="/u/102129653793910296719/" id="projects-dropdown" onclick="return false;"
 ><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="/u/102129653793910296719/" onclick="_CS_click('/gb/ph/profile');"
 title="Profile, Updates, and Settings"
 ><u>Profile</u></a>
 | <a href="https://www.google.com/accounts/Logout?continue=https%3A%2F%2Fcode.google.com%2Fhosting%2Fsearch%3Fq%3Dlabel%3Abiologicalpathways"
 onclick="_CS_click('/gb/ph/signout');"
 ><u>Sign out</u></a>

 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>


 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 >
 <tr style="height: 58px;">



 <td id="plogo">
 <link itemprop="url" href="">
 <a href="/">

 <img src="https://ssl.gstatic.com/codesite/ph/images/code_small.png" alt="Logo" itemprop="image">

 </a>
 </td>

 <td style="padding-left: 0.5em">

 </td>
 <td style="white-space:nowrap;">

 <form action="/hosting/search">
 <input size="40" name="q" value="label:biologicalpathways" type="text">

 <input type="submit" name="projectsearch" value="Search projects" >
 </form>

 </tr>
 </table>

</div>



<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"

>






<script type="text/javascript">
 function starClick(el, project_name) {
 _CS_toggleStar(
 el,
 {'scope': 'projects',
 'user': '_CURRENT_USER',
 'item': project_name
 },
 true,
 '',
 '',
 '',
 'star_count-' + project_name,
 'plural-' + project_name);
 }
</script>
<table class="mainhdr" width="100%" border="0" cellspacing="0" cellpadding="0">
<tr><td width="100%"></td>

 <td style="padding:3px 1em; font-weight:normal; white-space:nowrap;">
 Results 1 - 4 of 4
 </td>

</tr>
</table>
<br/>
<div id="serp">



<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/chibe/">
 <img style="max-width:50px;max-height:55px"

 src="/p/chibe/logo?cct=1402423200.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 1)"
 href="/p/chibe/" style="font-size:medium">chibe &nbsp;- &nbsp;Chisio BioPAX Editor</a>

 <img width="15" height="15" id="star_img"
 src="https://ssl.gstatic.com/codesite/ph/images/star_off.gif"
 style="cursor:pointer"

 onclick="starClick(this, 'chibe');"/>

 <br/>

 <span style="color:#676767">
 Updated: Last 30 days
 &nbsp;-&nbsp;
 Stars: <span id="star_count-chibe">6</span>
 </span><br/>




Chisio BioPAX Editor, or simply ChiBE, is a free editing and visualization tool for pathway models represented by the  BioPAX format, using  SBGN Process Description Language, based on  Chisio.
...<br/>

 <span class="labels">


 <a href="/hosting/search?q=label:Biology">Biology</a>,

 <a href="/hosting/search?q=label:Bioinformatics">Bioinformatics</a>,

 <a href="/hosting/search?q=label:BiologicalPathways">BiologicalPathways</a>,

 <a href="/hosting/search?q=label:BioPAX">BioPAX</a>,

 <a href="/hosting/search?q=label:SBGN">SBGN</a>,

 <a href="/hosting/search?q=label:GraphVisualization">GraphVisualization</a>,

 <a href="/hosting/search?q=label:Academic">Academic</a>


 </span>
 </td>
 </tr>
</table>



<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/mutation-to-expression/">
 <img style="max-width:50px;max-height:55px"

 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 2)"
 href="/p/mutation-to-expression/" style="font-size:medium">mutation-to-expression &nbsp;- &nbsp;For finding gene mutations that affect downstream expression</a>

 <img width="15" height="15" id="star_img"
 src="https://ssl.gstatic.com/codesite/ph/images/star_off.gif"
 style="cursor:pointer"

 onclick="starClick(this, 'mutation-to-expression');"/>

 <br/>

 <span style="color:#676767">
 Updated: Earlier this year
 &nbsp;-&nbsp;
 Stars: <span id="star_count-mutation-to-expression">1</span>
 </span><br/>


 This software statistically links genomic mutations of genes to the expression changes at their signaling downstream. Below is a sample influence network linking ESR1 overexpression to mutations of it...<br/>

 <span class="labels">


 <a href="/hosting/search?q=label:Bioinformatics">Bioinformatics</a>,

 <a href="/hosting/search?q=label:SystemsBiology">SystemsBiology</a>,

 <a href="/hosting/search?q=label:Biology">Biology</a>,

 <a href="/hosting/search?q=label:BiologicalPathways">BiologicalPathways</a>,

 <a href="/hosting/search?q=label:Genomics">Genomics</a>,

 <a href="/hosting/search?q=label:Mutation">Mutation</a>,

 <a href="/hosting/search?q=label:GeneExpression">GeneExpression</a>


 </span>
 </td>
 </tr>
</table>



<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/mutex/">
 <img style="max-width:50px;max-height:55px"

 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 3)"
 href="/p/mutex/" style="font-size:medium">mutex &nbsp;- &nbsp;Mutual exclusivity analysis of genomic alterations</a>

 <img width="15" height="15" id="star_img"
 src="https://ssl.gstatic.com/codesite/ph/images/star_off.gif"
 style="cursor:pointer"

 onclick="starClick(this, 'mutex');"/>

 <br/>

 <span style="color:#676767">
 Updated: Earlier this year
 &nbsp;-&nbsp;
 Stars: <span id="star_count-mutex">1</span>
 </span><br/>


 This project aims to discover driver alterations in genomic datasets by detecting mutual exclusion (mutex) patterns in alterations of groups of genes with a common downstream effect. Below are example...<br/>

 <span class="labels">


 <a href="/hosting/search?q=label:Bioinformatics">Bioinformatics</a>,

 <a href="/hosting/search?q=label:Genomics">Genomics</a>,

 <a href="/hosting/search?q=label:Cancer">Cancer</a>,

 <a href="/hosting/search?q=label:Algorithm">Algorithm</a>,

 <a href="/hosting/search?q=label:Analysis">Analysis</a>,

 <a href="/hosting/search?q=label:Mutualexclusivity">Mutualexclusivity</a>,

 <a href="/hosting/search?q=label:Mutation">Mutation</a>,

 <a href="/hosting/search?q=label:Copynumberalteration">Copynumberalteration</a>,

 <a href="/hosting/search?q=label:Geneexpression">Geneexpression</a>,

 <a href="/hosting/search?q=label:Biologicalpathways">Biologicalpathways</a>,

 <a href="/hosting/search?q=label:Signalingpathways">Signalingpathways</a>


 </span>
 </td>
 </tr>
</table>



<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/pcviz/">
 <img style="max-width:50px;max-height:55px"

 src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 4)"
 href="/p/pcviz/" style="font-size:medium">pcviz &nbsp;- &nbsp;Network Visualization for Pathway Commons</a>

 <img width="15" height="15" id="star_img"
 src="https://ssl.gstatic.com/codesite/ph/images/star_off.gif"
 style="cursor:pointer"

 onclick="starClick(this, 'pcviz');"/>

 <br/>


 A web-based network visualization software for the Pathway Commons database.<br/>

 <span class="labels">


 <a href="/hosting/search?q=label:Academic">Academic</a>,

 <a href="/hosting/search?q=label:Bioinformatics">Bioinformatics</a>,

 <a href="/hosting/search?q=label:Biology">Biology</a>,

 <a href="/hosting/search?q=label:Networks">Networks</a>,

 <a href="/hosting/search?q=label:BiologicalPathways">BiologicalPathways</a>,

 <a href="/hosting/search?q=label:ComputationalBiology">ComputationalBiology</a>


 </span>
 </td>
 </tr>
</table>


 </div>
 <div style="clear:both">&nbsp;</div>


<div style="text-align:center; font-size:medium; white-space:nowrap; margin-top:1em;">



</div>
<br/>
<br/>
<div style="background:#F0F7F9; border-top:1px solid #6B90DA; border-bottom:1px solid #6B90DA; text-align:center;">
 <br/>
 <form action="/hosting/search">
 <input type="text" size="30" name="q" value="label:biologicalpathways">
 <input type="hidden" name="mode" value="">
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 <br/>
</div>
<script type="text/javascript">
function clk(el, cd) {
 if (document.images) {
 (new Image()).src = "/hosting/search?sa=T&ct=res&cd=" + cd + "&url="
 + escape(el.href) + "&q=label%3Abiologicalpathways";
 }
 return true;
}
</script>



 <script type="text/javascript" src="https://ssl.gstatic.com/codesite/ph/3374407039186159233/js/ph_core.js"></script>




</div>

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>






 </body>
</html>

