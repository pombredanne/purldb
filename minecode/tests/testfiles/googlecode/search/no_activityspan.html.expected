[
  [
    "http://code.google.com/hosting/search?q=label:academic", 
    "http://code.google.com/hosting/search?q=label:algorithm", 
    "http://code.google.com/hosting/search?q=label:analysis", 
    "http://code.google.com/hosting/search?q=label:bioinformatics", 
    "http://code.google.com/hosting/search?q=label:biologicalpathways", 
    "http://code.google.com/hosting/search?q=label:biology", 
    "http://code.google.com/hosting/search?q=label:biopax", 
    "http://code.google.com/hosting/search?q=label:cancer", 
    "http://code.google.com/hosting/search?q=label:computationalbiology", 
    "http://code.google.com/hosting/search?q=label:copynumberalteration", 
    "http://code.google.com/hosting/search?q=label:geneexpression", 
    "http://code.google.com/hosting/search?q=label:genomics", 
    "http://code.google.com/hosting/search?q=label:graphvisualization", 
    "http://code.google.com/hosting/search?q=label:mutation", 
    "http://code.google.com/hosting/search?q=label:mutualexclusivity", 
    "http://code.google.com/hosting/search?q=label:networks", 
    "http://code.google.com/hosting/search?q=label:sbgn", 
    "http://code.google.com/hosting/search?q=label:signalingpathways", 
    "http://code.google.com/hosting/search?q=label:systemsbiology"
  ], 
  [
    {
      "url": "http://code.google.com/p/chibe/", 
      "last_updated": "2014-01-03T00:00:00+00:00", 
      "stars": 6
    }, 
    {
      "url": "http://code.google.com/p/mutation-to-expression/", 
      "last_updated": "2014-01-01T00:00:00+00:00", 
      "stars": 1
    }, 
    {
      "url": "http://code.google.com/p/mutex/", 
      "last_updated": "2014-01-01T00:00:00+00:00", 
      "stars": 1
    }, 
    {
      "url": "http://code.google.com/p/pcviz/", 
      "last_updated": null, 
      "stars": 0
    }
  ]
]